test/spec/provider/bpmn/CompensationProps.spec.js

```diff
@@ -14,7 +14,7 @@ import {
 } from 'min-dom';
 
 import CoreModule from 'bpmn-js/lib/core';
-import SelectionModule from 'diagram-js/lib/features/selection';
+import SelectEntryionModule from 'diagram-js/lib/features/selection';
```

Comment:
Same here. I think we can simply remove all these `SelectEntryionModule` changes.

Reply:
Ok fixed